### PR TITLE
Rewrite sentence

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Description

- Rewrite sentence on homepage:
  - Use the verb `click` instead of `press` (actually we should use a input-neutral verb!)
  - Change sentence order

## Checklist

- [x] I’ve added tests to confirm my change works.
- [x] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Review by sosukesuzuki — COMMENTED** [review]
I don't know how natural these English sentences are to me...

File: website/pages/en/index.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ const TldrSection = ({ language }) => (
         <div style={{ display: "flex", flexDirection: "column" }}>
           <h2>Why?</h2>
           <ul style={{ flex: "1" }}>
-            <li>You press save and code is formatted</li>
+            <li>Click save and your code is formatted</li>
```
Comment (HonkingGoose) [inline website/pages/en/index.js:99]: The [Microsoft Style guide](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui) has some good information about describing interactions with UI:

> Customers interact with products using different input methods: keyboard, mouse, touch, voice, and more. So use generic verbs that work with any input method. Don't use input-specific verbs, such as _click_ or _swipe_. Instead, use the verbs listed here.

The document listed above also has a table of good verbs, but none of these fit the Prettier homepage... 😞 

Based on their recommendation, my suggestion to use `click` doesn't fix the problem. 😄 So how about using `save` as the verb? 

```suggestion
            <li>Formats code when saving files</li>
```

**Comment by kachkaev** [comment]
👋 @HonkingGoose! Which practical problem are you trying to address?

**Comment by HonkingGoose** [comment]
## Intro

Hi @kachkaev! 👋

Good question! Let's look at the verbs associated with some input methods:

## Input methods and verbs

| Input method | Verb(s) |
| :--- | :--- |
| mouse/trackball | `click` |
| keyboard | `type` |
| voice commands | `speak` |
| touchpad/table/smartphone | `swipe` / `tap` |
| eyetracking | `look` |
| controller | `move joystick`, `press trigger`, `press button` |

See how each input method has it's own verb?

## Benefits of using input-neutral verbs

Using an input-neutral verb like `select` has these benefits:

- The verb works for each input device, the user already knows how to use their input device to select things
- We use accessible language, as we're assuming that people use a keyboard or mouse
- It's easier to use one single verb, instead of figuring out if the user is `typing`/`clicking`/`swiping`/`tapping` something

## Summary

I'm not solving an practical problem with this PR. The goal here is to use accessible language, and simplify writing about UI interactions. 😉

**Comment by apmeehan** [comment]
@HonkingGoose While I'm a fan of input-device-agnostic language, "_Formats code when saving files_" feels to me like it's missing some of the immediacy and... "snappiness" of the original :)

What about something like "_Save your file and your code is formatted_", or just "_Your code is formatted on save_"?

File: website/pages/en/index.js (hunk 1 above)
Comment (HonkingGoose) [inline website/pages/en/index.js:99]:
```suggestion
            <li>Your code is formatted on save</li>
```

**Review by HonkingGoose — COMMENTED** [review]
I like the suggestion from @apmeehan best.

**Comment by fisker** [comment]
"Your code is formatted on save" souds good.

**Comment by fisker** [comment]
"Your code is formatted on save" souds good.

**Review by fisker — APPROVED** [review]

**Comment by fisker** [comment]
Yes, need more review.

**Review by PraveenNanda124 — COMMENTED** [review]
Good work

**Comment by HonkingGoose** [comment]
Thank you all for reviewing my PR, and for merging it! 😄 ❤️

**Pull request merged**